workspaces/liferay-sample-workspace/client-extensions/liferay-sample-theme-css-4/src/css/_custom.scss

```diff
@@ -0,0 +1,5 @@
+@import 'sassy-inputs/sass/main';
+
+body {
+	font-family: Arial, Helvetica, sans-serif;
+}
```

Comment:
Why don't we make this css more obvious when it get's applied:

```
.control-menu {
	background: linear-gradient(105deg, #003f5b, #2b4b7d, #5f5195, #98509d, #cc4c91, #f25375, #ff6f4e, #ff9913) !important;
}

.control-menu-level-1-heading {
	color: #fff !important;
}
```

Let's also remove the sassy-inputs from here and the package.json. No need.

Reply:
Better! Thanks